mod/primitives/pkg/encoding/hex/string.go

```diff
@@ -108,19 +106,9 @@ func FromJSONString[B ~[]byte](b B) String {
 	return NewString(bytes.Trim(b, "\""))
 }
 
-// Has0xPrefix returns true if s has a 0x prefix.
-func (s String) Has0xPrefix() bool {
-	return has0xPrefix[string](string(s))
-}
-
-// IsEmpty returns true if s is empty.
-func (s String) IsEmpty() bool {
-	return len(s) == 0
-}
```

Comment:
exported just for tests and always checked together. Consolidated into `IsValidHex`